### DISCUSSION TITLE
Animated points

### DISCUSSION THREAD

**Discussion by TemMax** [patrykandpatrick/vico#831] [Questions] [answered]
### Question

Is it possible to make the point animated? For example, it can be used to create a "wave" effect (example in attachments). I've tried, but there is no `invalidate` function or something else for the custom `Component` instance.
I've attached an example here 🙌

https://github.com/user-attachments/assets/f58a6ef9-9ee1-46c8-af5a-f1223c50d49b

### Vico version(s)

2.0.0-alpha.27

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, @TemMax. Unfortunately, this is not possible yet. We will follow up to this topic under #878.

**Comment by TemMax** [comment]
Got it. Thanks a lot!
